core/src/main/java/tech/pegasys/web3signer/core/multikey/SignerLoader.java

```diff
@@ -84,4 +118,15 @@ private static void renderException(final Throwable t, final String filename) {
         ExceptionUtils.getRootCauseMessage(t));
     LOG.debug(ExceptionUtils.getStackTrace(t));
   }
+
+  private static int numberOfThreads() {
+    int defaultNumberOfThreads = Runtime.getRuntime().availableProcessors() - 1;
+    // we only want to use 10 threads at max
+    if (defaultNumberOfThreads >= 10) {
```

Comment:
10 should be a constant somewhere rather than a magic number.

Reply:
refactored to a constant.